examples/yargs-cli/package-lock.json

```diff
@@ -9581,9 +9495,7 @@
       "resolved": "https://registry.npmjs.org/jsonparse/-/jsonparse-1.3.1.tgz",
       "integrity": "sha512-POQXvpdL69+CluYsillJ7SUhKvytYjW9vG/GKpnf+xP8UWgYEM/RaMzHHofbALDiKbbP1W8UEYmgGl39WkPZsg==",
       "dev": true,
-      "engines": [
-        "node >= 0.2.0"
-      ]
+      "engines": ["node >= 0.2.0"]
```

Comment:
just checking, are we sure we want `package-lock` to be linted/fixed? there will be an eternal diff after each `npm install` :thinking: